### DISCUSSION TITLE
Help with LTSP

### DISCUSSION THREAD

**Discussion by N0odlez** [ltsp/ltsp#177] [Q&A]
I'm going to be converting old hardware to use linux to connect to our new RDS 2019 farm next month. I was wondering if you would be willing to help me get this up and running and in return you'll be able to use us as a test bed on over 400 devices. I would be greatful on some more infomation on how to deploy this on a large scale and also how we can get freerdp installed with pre configured options.

Many thanks

**Comment by alkisg** [comment]
Hi @AzureusNation,

I moved this issue to the community issue tracker, as it doesn't directly involve LTSP code changes.

At this point I have deployed the new LTSP in enough schools here and there's no need anymore for test beds. Additionally, we're not using freerdp so personally I don't have much interest about it.

So, I could just give you a few pointers here, or you could hire me to do this as a job. In that case, please send me an email to alkisg at gmail.

**Comment by N0odlez** [comment]
I guess the main question is, what does LTSP actually achieve?

Is it just a linux repackager and creates an installer?

If so, is it just a case of me editing the linux install in a VM and then using ltsp to package that up as it is?

**Comment by alkisg** [comment]
It takes a normal (or minimal, or chroot, or VM) installation and creates something like a "live cd image" out of it for the clients to netboot from it. The clients are diskless, they get that image from the network. It's one read only image for all the clients.

Then the clients boot from it, they may authenticate against the server via SSH or LDAP, they may use the server /home via NFS or SSHFS, and of course they can run any program like xfreerdp or whatever.

So in your case you'd do a minimal installation, then you'd use autologin to login to a session, and that session would just run xfreerdp to authenticate and connect to your server.

**Comment by N0odlez** [comment]
Fantastic, last thing really, is there a way of making a Linux installer so I can distribute this to install on to SSDs

**Comment by alkisg** [comment]
LTSP doesn't have an installer, but if you use a VM instead of a chroot, you can just `dd` it to local disks while they're netbooted.
